index.js

```diff
@@ -1,34 +1,34 @@
-const STRINGTYPE = Buffer.from([6, 0])
-const BOOLTYPE = Buffer.from([6, 1])
-const BOOLTRUE = Buffer.from([1])
-const UNDEFINEDTYPE = Buffer.from([6, 2])
-
-const FEEDTYPE = Buffer.from([0])
-const CLASSICFEEDTYPE = Buffer.from([0, 0])
-const GGFEEDTYPE = Buffer.from([0, 1])
-const BBFEEDTYPE = Buffer.from([0, 3])
-
-const MSGTYPE = Buffer.from([1])
-const CLASSICMSGTYPE = Buffer.from([1, 0])
-const GGMSGTYPE = Buffer.from([1, 1])
-const BBMSGTYPE = Buffer.from([1, 4])
+const STRING_TYPE = Buffer.from([6, 0])
+const BOOL_TYPE = Buffer.from([6, 1])
+const BOOL_TRUE = Buffer.from([1])
+const UNDEFINED_TYPE = Buffer.from([6, 2])
+
+const FEED_TYPE = Buffer.from([0])
+const CLASSIC_FEED_TYPE = Buffer.from([0, 0])
+const GG_FEED_TYPE = Buffer.from([0, 1])
+const BB_FEED_TYPE = Buffer.from([0, 3])
+
+const MSG_TYPE = Buffer.from([1])
+const CLASSIC_MSG_TYPE = Buffer.from([1, 0])
+const GG_MSG_TYPE = Buffer.from([1, 1])
+const BB_MSG_TYPE = Buffer.from([1, 4])
 const nullMsgBytes = Buffer.alloc(32)
```

Comment:
Any reason for leaving this as camel case? Might be more consistent as `NULL_MSG_BYTES`?

Reply:
Yeah, I thought about that for a while too, I'm not sure it belongs to the same category as the others, and I didn't author this code so I felt maybe we should keep it like this. We can change it, but perhaps in another PR.